lib/services/template-service.ts

```diff
@@ -83,37 +82,26 @@ export class TemplateService implements ITemplateService {
     }
 
     getTemplates(): Promise<Array<any>> {
-        let templateDetails: Array<any> = [];
         const promises: Array<any> = [];
 
         return new Promise((resolve, reject) => {
-            templateCache.get("templateDetails", (error: any, value: any) => {
+            this.templateCache.get("templateDetails", (error: any, value: any) => {
                 if (!error) {
```

Comment:
What happens if there is error here? We are not handling this case explicitly.

Reply:
Good question!
@borisKarastanev ?